tests/validation/lib/aws.py

```
@@ -497,3 +497,28 @@ def decrypt_windows_password(self, instance_id):
                 password = rsa.decrypt(password,priv).decode('utf-8')
 
         return password
+
+    def get_ebs_volumes(self, provider_node_id):
+        node_filter = [{
+            'Name': 'attachment.instance-id', 'Values': [provider_node_id]}]
+        try:
+            response = self._client.describe_volumes(Filters=node_filter)
+            volumes = response.get('Volumes', [])
+            return volumes
+        except Boto3Error as e:
```

Runtime error is not handled in except, we can handle it as below.
```except (Boto3Error, RuntimeError) as e:```